### PR TITLE
WORKSPACE put and post with array of projects.

### PR DESCRIPTION
**Pull request by 666nonak666**
test api workspace and method to change Map to Json format.

### CODE REVIEW

File: src/test/java/org/fundacionjala/pivotal/cucumber/hooks/Hooks.java [hunk 1]
```diff
@@ -51,7 +51,9 @@ public void resetworkspace() {
       while (iter.hasNext()) {
           StoreVariables variable = iter.next();
           if (variable.getName().contains("Workspace")) {
-              RequestManager.delete(Utils.buildEndpoint(String.format("/my/workspace/[%s.id]", variable.getName())));
+              System.out.println(Utils.buildEndpoint(String.format("/my/workspaces/[%s.id]", variable.getName())));
```
Comment (rubercuellar) [inline src/test/java/org/fundacionjala/pivotal/cucumber/hooks/Hooks.java]: Remove this line

File: src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java [hunk 2]
```diff
@@ -69,4 +74,68 @@ public static String buildEndpoint(final String endpoint) {
         return "";
     }
 
+    /**
+     * convert map to json.
+     *
+     * @param body map.
+     * @return element in Json.
+     */
+    public static String convertJson(final Map<String, String> body) {
```
Comment (rubercuellar) [inline src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java:85]: Is there no library to convert jsons directly?

File: src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java [hunk 3]
```diff
@@ -69,4 +74,68 @@ public static String buildEndpoint(final String endpoint) {
...
+     * draw value of String.
+     *
+     * @param array String where is the value.
+     * @return string of value.
+     */
+    private static String dataValue(final String array) {
+
+        String[] point = array.split(",");
+        for (int i = 0; i < point.length; i++) {
+            System.out.println(point[i]);
```
Comment (rubercuellar) [inline src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java]: Remove this line

File: src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java [hunk 4]
```diff
@@ -69,4 +74,68 @@ public static String buildEndpoint(final String endpoint) {
...
+            point[i] = buildEndpoint(point[i]);
+            System.out.println(point[i]);
```
Comment (rubercuellar) [inline src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java]: Remove this line

File: src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java [hunk 5]
```diff
@@ -69,4 +74,68 @@ public static String buildEndpoint(final String endpoint) {
...
+
+    }
+
+    /**
+     * verify if had , in.
+     *
+     * @param body map where search.
+     * @return boolean.
+     */
+    public static boolean verifyArray(final Map<String, String> body) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java:124]: This method can be refactor in a single line of code using stream.
@AT-03/at-03 
Please review it.

**Pull request closed without merging**